src/main/java/uk/gov/hmcts/divorce/divorcecase/model/AlternativeServiceOutcome.java

```diff
@@ -0,0 +1,134 @@
+package uk.gov.hmcts.divorce.divorcecase.model;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.ccd.sdk.api.CCD;
+import uk.gov.hmcts.ccd.sdk.type.YesOrNo;
+import uk.gov.hmcts.divorce.document.model.DivorceDocument;
+
+import java.time.LocalDate;
+
+import static uk.gov.hmcts.ccd.sdk.type.FieldType.Email;
+import static uk.gov.hmcts.ccd.sdk.type.FieldType.FixedList;
+import static uk.gov.hmcts.ccd.sdk.type.FieldType.TextArea;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class AlternativeServiceOutcome {
+
+    @CCD(
+        label = "Application date",
+        displayOrder = "1"
```

Comment:
should these be integers?

Reply:
Yeah. I initially created the displayOrder attribute as an Integer in the CCD Interface class, but I couldn't initially work out how to get the default value in the Interface to be null. I'm happy to go back and change it, or come up with another solution if we cannot use null defaults.

Reply:
assuming it defaults to 0 we could treat that is null

Reply:
Ok - will implement this solution